### PR TITLE
CI: collection-2021-1.2

### PR DESCRIPTION
**Pull request by mrakitin**
Uses https://github.com/NSLS-II/profile-collection-ci/pull/19.

### CODE REVIEW

**Comment by mrakitin** [comment]
https://dev.azure.com/nsls2/profile_collections/_build/results?buildId=1888&view=results is displaying the current results after rebasing https://github.com/NSLS-II/profile-collection-ci/pull/19 on main to have the fixes for https://github.com/NSLS-II/profile-collection-ci/commit/fa6b7c29bd9ae23fcbfb114646d5f9b9facf0d24 and https://github.com/NSLS-II/profile-collection-ci/commit/ef6513119bf5c999bd345fa865112b3dd44f6814.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -3,10 +3,15 @@ resources:
     - repository: templates
       type: github
       name: NSLS-II/profile-collection-ci
-      ref: refs/heads/main
+      # TODO: change it back to 'refs/heads/main' once
+      # https://github.com/NSLS-II/profile-collection-ci/pull/19 is merged.
+      ref: refs/heads/add-collection-2021-1.2
```
Comment (mrakitin) [inline azure-pipelines.yml:8]: Note: do not forget this TODO after https://github.com/NSLS-II/profile-collection-ci/pull/19 is merged to `main`.

**Review by AbbyGi — APPROVED** [review]
Looks good.

**Comment by mrakitin** [comment]
Power-cycled to restart CI.

**Pull request merged**